modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -1575,6 +1575,57 @@ components:
             required:
                 - name
             type: object
+        DocumentShortcut:
+            description:
+                Represents a document shortcut.
+            properties:
+                assetLibraryKey:
```

Comment:
@SamZiemer, the actions are missing here. You calculate them here: https://github.com/liferay-content-management/liferay-portal/pull/5444/files#diff-f84a35e8e7a7d45e24abafbeeeac33db373a7ae0e3ce9a07ba65290d98a1680cR45 but they are not used in the DTO converter here https://github.com/liferay-content-management/liferay-portal/pull/5444/files#diff-635b268087903e836f93733a12dbc85ddd4bce95b4f9b0dcf6f49104ac867f07R47